components/ProjectLink.js

```diff
@@ -0,0 +1,17 @@
+const ProjectLink = ({ title, url, className }) => (
+  <a
+    href={url}
+    target="_blank"
```

Comment:
this causes the link to open in a separate window, instead of navigating away from your site

Reply:
Oh oops, I meant to add this when I was building that part but must have forgotten, will definitely add it in, thanks for catching!